src/main/java/uk/gov/hmcts/reform/sscs/client/CaseWorkerRefDataApi.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.sscs.client;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import java.util.List;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.sscs.model.CaseWorkersProfileCreationRequest;
+import uk.gov.hmcts.reform.sscs.model.ServiceRoleMapping;
+import uk.gov.hmcts.reform.sscs.model.UserRequest;
+
+
+@FeignClient(
+    name = "caseworker-ref-data-api",
+    url = "${caseworker-ref.api.url}"
+)
+public interface CaseWorkerRefDataApi {
+    String SERVICE_AUTHORIZATION = "serviceAuthorization";

```

Comment:
Should this have an access modifier? (Same as below)

Reply:
access modifiers are not needed for interface methods or variables. 
this has also been done in the existing interface `RefDataApi.java`

Reply:
Ahaha, wow. I completely missed it was an interface

Reply:
No as it's an interface - I suggest we should refactor these clients at a later stage to put this string in a constants class as we all have it in the interfaces.